sql-cli/sql_cli/__init__.py

```diff
@@ -1,13 +1,18 @@
 import importlib.metadata
 import os
 
+import airflow.providers.sqlite.hooks.sqlite
 import typer.rich_utils
 
+from sql_cli.hooks.sqlite import SqliteHook
 from sql_cli.utils.rich import rich_format_error
 
 # We monkey-patch rich_format_error to make it environment aware
```

Comment:
Does environment-aware mean running the plain `flow` command or running it from `Astro CLI`? If that's the case, perhaps we could be more explicit about it.

Reply:
Right - "environment" is confusing here. It means running it from `Astro CLI`.